README.md

```diff
@@ -49,11 +55,72 @@ This will give you an admin user with archipelago as password (!change this if r
 
-# Step 4: Optional, create a demo user using drush 
+## Step 4: Optional, create a demo user using drush 
 ```Shell
 docker exec -ti esmero-web bash -c 'drush ucrt demo --password="demo"; drush urol metadata_pro "demo"'
 ```
 
 Open your most loved Web Browser and point it to http://localhost:8001
 
 See any errors? Let us know.
+
+# Updating an existing Archipelago pulled from here (minio branch)
+
+## Step A
+
+- A. If you made local changes to your content, added files, setup new formatters or changed blocks around means you are amazing and all will be fine. This skeleton project ignores all of that. Skip to [step B](#step-b) 
+
+- B. If you installed new Drupal modules, added new dependencies to composer packages, did some manual docker-compose.yml editing an/or went creative and made mayor changes and you fear you will loose them (yes you could), you are also amazing but you should do the following fist.
+
+Create a new Local Branch and move all your changes to that one. 
+
+```Shell
+git stash
+git checkout -b minio-mylocalchanges
+git stash apply
+```
+
+This won't actually commit your changes to the new branch but the changes will be there for you to decide how to deal with them. You could do a `git commit -am "All my changes in single commit"` to make sure they persist. Not saying making a single commit for 2999 file changes is the way! Just that it is "a" way.
+
+This will preserve your local changes in that branch. 
+
+## Step B
+
+```Shell
+git checkout minio
+git pull
+```
+
+Look at the output of that command and look for larger changes? Did you loose a file? Depending on when you did the last `git pull` a file that could got lost is `docker-compose.yml`. We recently (to avoid you having to do this) added it to `.gitignore`. But if your `.gitignore` still has no entry for it, you will loose it. No harm really since you are trying to keep your vanilla docker deployment in shape here. If it was removed (sorry) do.
+
+```Shell
+cp docker-compose-minio.yml docker-compose.yml
+```
+
+It was always a copy, so doing that will keep your docker-compose working as before.
+
+If your `docker-compose.yml` was highly customized, fancy and totally diverging from the original here we think its better to manage a paralell branch for that and manually bring the changes back. You can see your previous docker-compose.yml inside your new `minio-mylocalchanges` branch or reset the current to a previous commit hash if needed. (`git log` is your friend)
+
+## Step C
+
+Now make Drupal/compose aware of the updates you pulled
+
+```Shell
+docker exec -ti esmero-web bash -c "composer install"
+```
+
+That last command will use the updated `composer.lock` file to download/use exactly the same package versions and dependencies we have tested during development. Just in case and to see Archipelago is still in shape, clear your Drupal caches
+
+```Shell
+docker exec -ti esmero-web bash -c "cd web; drush cr"
```

Comment:
@DiegoPino I used 
`docker exec -ti esmero-web bash -c "drush cr"` 
without `cd web`
I don't know if that depend on my installation, just in case.

Reply:
Yes! you are right.
I guess it depends on the installation really and if drush was deployed globally?  I had some issues in the past calling drush outside the folder of the installation.  to be sure i just tested both with -d option

```SHELL
[preflight] Config paths: /var/www/html/vendor/drush/drush/drush.yml,/var/www/html/drush/drush.yml
 [preflight] Alias paths: /var/www/html/web/drush/sites,/var/www/html/drush/sites
 [preflight] Commandfile search paths: /var/www/html/vendor/drush/drush/src,/var/www/html/drush
 [bootstrap] Starting bootstrap to site [0.73 sec, 3.22 MB]
 [bootstrap] Drush bootstrap phase 2 [0.73 sec, 3.22 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.74 sec, 3.22 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.74 sec, 3.22 MB]
 [bootstrap] Try to bootstrap at phase 2 [0.76 sec, 3.29 MB]
 [bootstrap] Drush bootstrap phase: bootstrapDrupalRoot() [0.76 sec, 3.29 MB]
 [bootstrap] Change working directory to /var/www/html/web [0.76 sec, 3.29 MB]
 [bootstrap] Initialized Drupal 8.6.14 root directory at /var/www/html/web [0.76 sec, 3.29 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.76 sec, 3.29 MB]
 [bootstrap] Try to bootstrap at phase 2 [0.8 sec, 3.39 MB]
 [bootstrap] Drush bootstrap phase: bootstrapDrupalSite() [0.8 sec, 3.39 MB]
 [bootstrap] Initialized Drupal site default at sites/default [0.8 sec, 3.39 MB]
 [success] Cache rebuild complete. [22.14 sec, 91.55 MB]
````

Same output!

Will fix, thanks!